src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -441,7 +441,7 @@ g.test('minBindingSize')
       .combine('minBindingSize', [undefined, 4, 256])
```

Comment:
Maybe change to `[undefined, 8, 256]` or (better?) `[undefined, 4, 8, 256]`? Since a `minBindingSize` of 4 may result in zero size buffer, which is kind of special.

Reply:
Done